packages/ui/tailwind.config.js

```diff
@@ -0,0 +1,18 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+    content: ['./src/**/*.{ts,tsx,mdx}'],
+    darkMode: ['class', '[data-mode="dark"]'],
+    theme: {
+        extend: {
+            colors: {
+                success: '#12B76A',
+                warning: '#FDB022',
+                error: '#F04438',
+            },
+            fontFamily: {
+                poppins: ['Poppins', 'sans-serif'],
```

Comment:
I think there's a mistake somewhere - you're setting the styles to use Popping, but importing the Inter font in the stylesheets 🤔

Reply:
Thanks for catching, I've updated this